funnel/views/email.py

```diff
@@ -122,6 +122,23 @@ def send_password_reset_link(email, user, token):
     send_email(subject, [(user.fullname, email)], content)
 
 
+@signals.proposal_submitted.connect
+def send_email_for_proposal_submission(proposal):
+    for editor in proposal.project.active_editor_memberships:
+        send_email(
+            subject=_("{project} has received a new proposal submission").format(
```

Comment:
"New proposal in {project}" will make it easier to recognise it from the email inbox.